platforms/Windows/shared/swift.en-us.wxl

```diff
@@ -16,13 +16,21 @@
   <String Id="Sdk_ProductName_arm64" Value="Swift Windows SDK (ARM64)" />
   <String Id="Sdk_ProductName_amd64" Value="Swift Windows SDK (AMD64)" />
   <String Id="Sdk_ProductName_x86" Value="Swift Windows SDK (X86)" />
+  <String Id="AndroidSdk_ProductName_aarch64" Value="Swift Android SDK (Arm64)" />
```

Comment:
The suffixes are inconsistent with elsewhere. Why not use the same? Is it because we're formatting the string elsewhere?

Also the casing of Arm64 is different.